example/build.gradle

```diff
@@ -47,4 +48,25 @@ dependencies {
     reactor_3_3_x 'io.projectreactor:reactor-core:3.3.0.BUILD-SNAPSHOT', {
         changing = true
     }
+
+    jmh project(path: ":agent", configuration: 'shadow')
+    //Uncomment for benchmark execution by Idea plugin
+//    jmh "org.openjdk.jmh:jmh-generator-annprocess:1.21"
+}
+
+jmh {
+    if (project.hasProperty("jmh.simpleTravisMode")) { //temporary for JHM sanity testing in Travis
+        include = 'reactor\\.blockhound\\.BlockHoundDummyBenchmark.*'
+        warmupIterations = 1
+        iterations = 3
+        fork = 1
+    } else {
+        include = 'reactor\\.blockhound\\.BlockHound.*Benchmark.*'
+        warmupIterations = 3
+        iterations = 5
+        fork = 3
+    }
+    resultsFile = project.file("${project.buildDir}/reports/jmh/results${System.currentTimeMillis()}.txt")
```

Comment:
it will make it harder to work with (a lot of files created), let's remove the timestamp from the file's name

Reply:
It depends what you plan to achieve. But in the end - the stable state - one file is ok.